learn/glossary.md

```diff
@@ -19,28 +17,70 @@ There are two design patterns recommended for AAT usage:
 * **Optimizing for Safety:** __ A simple backend server that distributes signed AAT’s using the clients Pocket Account. Though this pattern is of higher effort for the Application, it provides the highest security guarantee around their AAT
 * **Optimizing for Performance:** Embed a token production system within the Application code. This guarantees the smoothest UX for the clients and easiest setup for the Application. However, reverse engineering a source code level token generator can be trivial if obfuscation methods are not applied. The upside to this approach is not having the need to have an additional component in the Application that generates the AAT dynamically while keeping the Application private key secure.
 
+## Application Burn Rate (ABR)
+
+Proposed mechanism that would burn developer's stakes based on application usage, and at a rate that offsets future inflation of the POKT cryptocurrency. Implementing ABR could help cap the total amount of POKT and promote network equilibrium where mint rate and burn rates are equal. For more information, see [Monetary Policy](economics/monetary-policy.md).
+
 ## Block Hash
 
-The SHA3-256 hash of a Block.
+SHA3-256 hash of a Block.
+
+## Endpoint
+
+A URL that can receive RPC requests to a blockchain. Pocket Network allows you to mint endpoints that can be used with any of the [supported blockchains](../supported-blockchains.md). An application can contain multiple endpoints (for multiple chains).
 
 ## Jail
 
-A Pocket Validator Node can be `Jailed` for 1 of 2 reasons:
+Situation where a Pocket node is denied the ability to serve relays.
+
+A node can be jailed for either of two reasons:
+
+1. Fails to produce `min_signed_per_window` amount of blocks over a `signed_blocks_window`. When jailed because of this reason, a node is `Slashed` a `slash_fraction_downtime`% of their `Stake`.
```

Comment:
No it does, because pretty soon we'll be able to have the values display alongside the `lingo`. With the values there, the `lingo` becomes really useful.

I just wanted to make it clear in case it wasn't that if you want to stake a validator, you need to be able to keep it online.